model/vm_host.rb

```diff
@@ -188,6 +188,14 @@ def download_boot_image(image_name, version:, custom_url: nil)
     Strand.create_with_id(schedule: Time.now, prog: "DownloadBootImage", label: "start", stack: [{subject_id: id, image_name: image_name, custom_url: custom_url, version: version}])
   end
 
+  # Introduced for downloading firmware via REPL.
+  def download_firmware(version_x64: nil, version_arm64: nil, sha256_x64: nil, sha256_arm64: nil)
```

Comment:
It looks like at a given time, only one pair of `version`, `sha256` would be used, right? If so, why do we get 4 parameters? Can we just get `version` and `sha256`? Then we wouldn't need to pick based on `arch`, it would be caller's responsibility to provide correct `version`/`sha256` pair (which kinda already is).

Reply:
The way it is right now can help to avoid mistakes when downloading firmware for a single host. This would work on x64 but fail on arm64.
`vmh.download_firmware(version_x64: "202402", sha256_x64: "somesha")`

When downloading new version for the whole fleet, it makes things a little easier, as you can do something like:
`VmHost.all.map { _1.download_firmware(version_x64: "202402", sha256_x64: "somesha", version_arm64: "202211", sha256_arm64: "anothersha")}`